site.properties

```diff
@@ -39,13 +39,13 @@ etlplugin.pipeline.s3.region = ap-southeast-2
 airflow.username = airflow
 airflow.password = airflow
 airflow.hostname = http://localhost:8080
+etl.s3.url = S3::mayank-test-databucket::ap-southeast-2::https://s3.ap-southeast-2.amazonaws.com
```

Comment:
Is there a chance , we can chnage this to a File location as default. 
We can chat for more context.